spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRegisterTableProcedure.java

```diff
@@ -55,10 +53,6 @@ public void dropTables() {
 
   @Test
   public void testRegisterTable() throws NoSuchTableException, ParseException {
```

Comment:
So we aren't actually adding any more tests here we are just removing this Assume clause?

Reply:
Yes. 
Already API level test is there for each catalog for register table.
Configuring other catalogs in this test case is hard (like Glue, REST). It is better to keep them as integration tests (may in in a follow up PR)

Now removing this check will make sure that this testcase runs for hadoop, hive, spark catalog.

Reply:
How do we Register a table in Hadoop? That doesn't sound possible

Reply:
`HadoopCatalog` also extends `BaseMetastoreCatalog`

So, when a register table is called. It makes a fresh commit to the Hadoop catalog too (file name will be as per hadoop catalog)
https://github.com/apache/iceberg/blob/master/core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java#L84

Reply:
OK Got it, Still not sure I think that was a good idea but it's already in.